packages/bar-chart/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [4.0.2](https://github.com/visa/visa-chart-components/tree/%40visa/bar-chart%404.0.2)(2020-12-22)
```

Comment:
There should be a space between `4.0.2` and `(2020-12-22)`. This requested change should apply to all the files.

Reply:
Good catch, addressed in aa9119384d27f715e14574c051421da15a6efdf8.